maxim/logger/together/client.py

```
@@ -99,13 +99,23 @@ def wrapper(self: ChatCompletions, *args: Any, **kwargs: Any):
             # Remove extra_headers from kwargs before sending to Together AI
             clean_kwargs = {k: v for k, v in kwargs.items() if k != "extra_headers"}
-            response = create_func(self, *args, **clean_kwargs)
+            try:
+                response = create_func(self, *args, **clean_kwargs)
+            except Exception as e:
+                if generation is not None:
+                    generation.error({"message": str(e)})
+                scribe().warning(
+                    f"[MaximSDK][TogetherInstrumentation] Error in generating content: {e}",
+                )
+                raise e
```

Ruff (0.12.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)